README.md

```diff
@@ -253,6 +254,34 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install data
 ```
 
+## HLS LSP Configuration
+
+haskell-language-server supports some forms of configuration.
```

Comment:
For what editor?

Reply:
Apparently so, [I took this information from the changelog](https://github.com/haskell/haskell-language-server/blob/master/ChangeLog.md#030)